CONTRIBUTING.md

```diff
@@ -116,11 +116,12 @@ In order to create a branch, either fork the repository or request access to the
 Be sure to link the issue you created earlier in the pull request.
 - In the bottom, you can select to either submit as a pull request or as a draft pull request.
 - Use a draft pull request first, so the community can already see the ongoing work.
+- In this stage, the review from parson takes place.
```

Comment:
Let's not name the service provider in the contribution guideline. It should be neutral and valid after the duration of this project.

Reply:
Agreed. Changed it to "service provider".